test/www/jxcore/test/thalicryptomanagertests.js

```diff
@@ -0,0 +1,61 @@
+require('./mockmobilegetdocpath');
+var test = require('tape');
+var cryptomanager = require('../thali/thalicryptomanager');
+
+
+test('successfully create a new pkcs12 file and return hash value', function (t) {
+  var errorMessage = null,
+      fileLocation = './pkcs12foldergood';
```

Comment:
:+1: 

I'm closing this comment because I have a separate comment on how setup/teardown should happen.